main.py

```diff
@@ -2,6 +2,44 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    create_genre = ["Western",
+                    "Action",
+                    "Dramma"]
+    for name in create_genre:
+        Genre.objects.create(
+            name=name
+        )
+
+    create_actors = [("George", "Klooney"),
```

Comment:
use consistence indentation throughout the code
You should split it like in 13-15 lines (add a line after [ and before ])